### ISSUE TITLE
Webcam testing accuracy ?

### ISSUE THREAD

**Issue by ghost** [JoshuaPiinRueyPan/ViolenceDetection#19]
In the snapshots taken from the camera, the model is incorrect. How can I improve accuracy?

**Comment by JoshuaPiinRueyPan** [comment]
Add more data.  To compare with the previous works, I use only the Hocket Dataset.  If you want to deploy it to the real work, you should add more training data.
  And if the training loss can't go down to the value as the Hocket Dataset, add more layer to the network.  Please refer to the src/net.  There are plenty of networks that I have tried in this project.  You can select the default one (i.e. the 'G2D19_P2OF_ResHB_1LSTM.py') and add more layers.
